src/pages/signin/LoginForm/LoginFormNarrow.js

```diff
@@ -105,31 +103,13 @@ class LoginFormNarrow extends React.Component {
                     />
                 </View>
 
-                <View style={[styles.mb6]}>
-                    <Text style={[styles.textLabel]}>
-                        With Expensify.cash, chat and payments are the same thing. Launching Summer 2021,
-                        {' '}
-                        join the waitlist to be first in line!
-                    </Text>
-                </View>
-
                 <View>
                     <Text style={[styles.textLabel, styles.textStrong, styles.mb1]}>
-                        Attention Open Source Developers:
+                        With Expensify.cash, chat and payments are the same thing.
                     </Text>
                     <Text style={[styles.textLabel]}>
-                        Enter your GitHub handle on the next page to skip the wait and join our dev-only beta;
-                        {' '}
-                        help build tomorrow and
-                        {' '}
-                        <Text
-                            style={[styles.link, styles.mx1]}
-                            onPress={() => openURLInNewTab(CONST.UPWORK_URL)}
-                        >
-                            earn cash
-                        </Text>
-                        {' '}
-                        today!
+                        {'Money talks. And now that chat and payments are in one place, it\'s also easy. '}
```

Comment:
Does this need to be in quotes? Or can we just encode the apos?

Reply:
We can encode the apos if we want and then do a space like the one i removed

Reply:
Personally, I think it is cleaner to do it with an encoded apos and an explict space.

Reply:
Sounds good I'll fix it!

Reply:
Updated